### ISSUE TITLE
Missing parentheses for sum quantifier in HTML

### ISSUE THREAD

**Issue by bmaclach** [JacquesCarette/Drasil#1080]
Drasil does not seem to be auto-generating parentheses where they are required in a sum quantifier in HTML. An example in SSP (Equation for IM1) is shown below, where there should be parentheses to show that the "_R<sub>n</sub>_" and "_T<sub>n</sub>_" are not included in the summation:

![image](https://user-images.githubusercontent.com/18634568/50358788-8766c200-0528-11e9-9882-8d4ba858e0ce.png)

Looking at #273, this was working in the past but must have broken at some point since then.

**Comment by smiths** [comment]
@bmaclach, I think this will be a fairly deep problem to solve, although @JacquesCarette will know better.  We have (correctly) moved the task of adding brackets to Drasil, but it means that we need our knowledge capture to be complete.  By order of operations, the products would be calculated first, and then the summation.  However, I don't know what the rules are for what symbols are in scope for the summation.  If it only applies to the next symbol (R_i and T_i, for the numerator and denominator, respectively), then the equation is ill formed.  If it applies to everything after the summation symbol, then we need to get R_n and T_n outside of the summation, as you said.  I think you are correct that we need parentheses.

**Comment by JacquesCarette** [comment]
Yes, it does seem like this is a bug. Though it shouldn't be too hard to fix, I think. The downside is that if you have a sum of two sums, it will likely add extra parens.
